examples/disaggregated/slurm/benchmark/start_worker.sh

```
@@ -50,7 +59,7 @@ fi
 #check if nsys_folder is provided
 if [ -z "${nsys_folder:-}" ]; then
     echo "nsys is not enabled, start normal flow"
-    trtllm-llmapi-launch trtllm-serve ${model_path} --host $(hostname) --port ${port} --extra_llm_api_options ${config_file}
+    trtllm-llmapi-launch ${numa_bind_cmd} trtllm-serve ${model_path} --host $(hostname) --port ${port} --extra_llm_api_options ${config_file}
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)